twitchpopups.htm

```diff
@@ -16,6 +16,20 @@
         // The emoji that surrounds the hotseat messages.
         const hotseatEmoji = '🔥';
 
+        // Start displaying timed messages as soon as the source is added? '!timed on', '!timed true' or '!timed start' will start them if not.
+        const timedMessagesAtStart = true;

```

Comment:
Makes more sense for this to be false by default I think.

Reply:
Yep, defo.. I've changed it to be off by default and added a couple of lines to the readme.